src/Arcus.WebApi.All/Arcus.WebApi.All.csproj

```diff
@@ -13,6 +13,7 @@
     <RepositoryUrl>https://github.com/arcus-azure/arcus.webapi</RepositoryUrl>
     <PackageIconUrl>https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png</PackageIconUrl>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <NoWarn>NU5125;NU5048</NoWarn>
```

Comment:
Instead of suppressing warnings for NU5048, can't we solve that in a better way ?
https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048

Same as for 5125
https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5125

Reply:
Yes, also thought about that, but this requires a rethinking of all the repo's. We should make sure that we do this all together. For now, I think we can leave it as-is.

Reply:
Added an issue for this: https://github.com/arcus-azure/arcus/issues/134